### PR TITLE
Disable flaky integration test

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**
To be fixed and re-enabled (see #25).

### CODE REVIEW

File: Build.mak [hunk 1]
```diff
@@ -26,6 +26,12 @@ TEST_FILTER_OUT += \
 TEST_FILTER_OUT += \
 	$C/test/collectd/main.d 
 
+# integration tests which are temporarily disabled due to flakiness
+# to be fixed and re-enabled
+TEST_FILTER_OUT += \
+	$C/test/unixsocket/main.d \
```
Comment (nemanja-boric-sociomantic) [inline Build.mak]: I suspect `unixlistener` is flaky as well (just it happened it didn't already fail), as it is using the same best-effort approach to synchronizing processes.

Comment (gavin-norman-sociomantic) [reply Build.mak]: Updated with `selectlistener` and `unixlistener` disabled too.

**Comment by nemanja-boric-sociomantic** [comment]
Should target `v2.5.x`.

**Comment by gavin-norman-sociomantic** [comment]
Updated to only disable the `signalfd` test, now we have a simple fix for the others (#27). Rebased on v2.5.x.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by nemanja-boric-sociomantic** [comment]
LGTM. I would wait for #27 to be merged first, as this kinda depends on that one.

**Pull request merged**